src/Neo.CLI/CLI/MainService.NEP17.cs

```diff
@@ -45,7 +44,7 @@ private void OnTransferCommand(UInt160 tokenHash, UInt160 to, decimal amount, UI
             Transaction tx;
             try
             {
-                tx = CurrentWallet.MakeTransaction(snapshot, new[]
+                tx = CurrentWallet!.MakeTransaction(snapshot, new[]
```

Comment:
With nullable enabled, `NoWallet` would be unnecessary.

Reply:
You shouldn't do `!` unless 100% sure its not going to be `null`. Set `tx` to `nullable` like `Transaction? tx;`. then check for null. That's the point of `nullable`. If you don't want it to be `nullable` than check return for `null`, than  use `!`. Also another point of `nullable` is for you to be checking for `null`.

Reply:
Also the reason why you are getting the warning for `nullable` is because your not doing `null` checking. For example `if (tx == null)`.

Reply:
With noWallet you can use !

Reply:
> With nullable enabled, `NoWallet` would be unnecessary.

How?